tools/local-network/send-mobilecoind-txs.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+##File to automate local network or release testing using mobilecoind & mobilecoin-d json
+
+## .json file with account keys in form {"view_private_key":"..", "spend_private_key":".."}
+
+## Amount to send
+AMOUNT=0.001
+## Location of keyfile with view_private_key and spend_private_key
+KEYFILE=""
+## Length to sleep between sending transactions
+SLEEP=1.5
+## Curl encoding
+enc="Content-Type: application/json"
+
+while [ "$1" != "" ]; do
+	case $1 in
+		-a | --amount)
+		  AMOUNT=$1
+		-k | --keyfile)
+		  KEYFILE=$1
+		-s | --sleep)
+		  SLEEP=$1
```

Comment:
IIUC these should consume the next arg
```suggestion
		-a | --amount)
		  shift
		  AMOUNT=$1
		-k | --keyfile)
		  shift
		  KEYFILE=$1
		-s | --sleep)
		  shift
		  SLEEP=$1
```

Reply:
Done

Reply:
What you have now will set `AMOUNT="-a"` and so on -- the loop already checks `$1`, so we need to consume it. You could `shift` after assigning `X=$2` (not `$1`)

Reply:
Correct! Fix'd, tested against the test network